docs/_documentations/remote-configuring.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: docs
+title: Configuring Codewind remote
+description: Configuring your environment to use Codewind remote
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing Cloud connection
+duration: 5 minutes
+permalink: remoteconfiguring
+type: document
+parent: installoncloud
+order: 1
+---
+
+## Configuring Codewind remote
+
+To securely configure Codewind remote, do the following steps.
+
+1. Optional: Create a namespace to install into by running the following command: `kubectl create namespace <your_namespace>`. If you do not create a new namespace, Codewind uses the current one.
```

Comment:
For OpenShift, the following needs to be added for pre-setup instructions:
```
oc adm policy add-scc-to-group anyuid system:serviceaccounts:<namespace>
oc adm policy add-scc-to-group privileged system:serviceaccounts:<namespace>
```

Reply:
Namespace no longer in the procedure following rewrite.